Web/Models/ViewPaymentV1ViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using Aiia.Sample.AiiaClient.Models;
+
+namespace Aiia.Sample.Models;
+
+public class ViewPaymentV1ViewModel
+{
+    public ViewPaymentV1ViewModel(object payment, PaymentType paymentType, PaymentReconciliationV1Response reconciliation)
+    {
+        Payment = payment;
+        PaymentType = paymentType;
+        Reconciliation = reconciliation;
+
+    }
+
+    public object Payment { get; set; }
+    public PaymentType PaymentType { get; set; }
+    public PayerTokenModel PayerToken { get; set; }
```

Comment:
could PayerToken be one of the arguments in the constructor? It is easy to forget to update it after creation IMO

Reply:
I guess the payer token is only relevant for the accept payments, so i understand if you dont want to have it by default